lms/djangoapps/teams/static/teams/js/spec/views/edit_team_spec.js

```diff
@@ -31,33 +38,57 @@ define([
 
                 var message = teamEditView.$('.wrapper-msg');
                 expect(message.hasClass('is-hidden')).toBeFalsy();
-                expect(message.find('.title').text().trim()).toBe("Your team could not be created!");
+                var actionMessage = teamAction === 'create' ? 'Your team could not be created!' : 'Your team could not be updated!';
```

Comment:
Are these visible in the UI? If yes, sorry if I missed this in some earlier story. We should change the exclamation marks to periods: "Your team could not be created." "Your team could not be updated."